test/plugins/commands.test.js

```diff
@@ -1091,6 +1095,70 @@ describe('command plugin test', () => {
             });
         });
 
+        it('creates commands tag if has good permission and tag does not exist with tag', () => {
+            commandTagFactoryMock.create.resolves(testCommandTagWithTag);
+            commandTagFactoryMock.get.onFirstCall().resolves(testCommandTag);
+            commandTagFactoryMock.get.onSecondCall().resolves(null);
+
+            options.payload = payloadTag;
+
+            return server.inject(options).then((reply) => {
+                const expectedLocation = {
+                    host: reply.request.headers.host,
+                    port: reply.request.headers.port,
+                    protocol: reply.request.server.info.protocol,
+                    pathname: `${options.url}/1`
+                };
+
+                assert.deepEqual(reply.result, hoek.merge({ id: 1 }, payloadTag));
+                assert.strictEqual(reply.headers.location, urlLib.format(expectedLocation));
+                assert.calledWith(commandFactoryMock.get, {
+                    namespace: 'screwdriver',
+                    name: 'test',
+                    version: '1.2.0'
+                });
+                assert.calledWith(commandTagFactoryMock.get, {
+                    namespace: 'screwdriver',
+                    name: 'test',
+                    tag: 'stable'
+                });
+                assert.calledWith(commandTagFactoryMock.create, {
+                    namespace: 'screwdriver',
+                    name: 'test',
+                    tag: 'stable',
+                    version: '1.2.0'
+                });
+                assert.calledTwice(commandTagFactoryMock.get);
+                assert.equal(reply.statusCode, 201);
+            });
+        });
+
+        it('update command tag if has good permission and tag exists with tag', () => {
```

Comment:
update**s**
if has -> if it has
what do you mean by `tag exists with tag` ? I think tags are attached to a version.

Reply:
> updates

fixed

> if has -> if it has

fixed

> what do you mean by tag exists with tag ? I think tags are attached to a version.

I changed the sentence a bit.
I write `with tag` because this test send tag information to API instead of version.